internal/fs/inode/file.go

```diff
@@ -178,38 +180,74 @@ func (f *FileInode) clobbered(ctx context.Context) (b bool, err error) {
 	return
 }
 
-// Ensure that f.content != nil
+// Helper method to open a reader
```

Comment:
This does not provide any information in addition to the function name `openReader`. You can move the line 188 here which is more informative about the reader.